bindings/nodejs/lib/types/wallet/event.ts

```diff
@@ -249,14 +249,14 @@ class PreparedTransactionProgress extends TransactionProgress {
 /**
  * A 'prepared transaction hash' progress.
  */
-class PreparedTransactionHashProgress extends TransactionProgress {
+class PreparedTransactionSigningHashProgress extends TransactionProgress {
     hash: HexEncodedString;
```

Comment:
I think you have renamed the field in rust so it needs to be the same to be parsed correctly